app/Models/Rule.php

```
@@ -0,0 +1,125 @@
+    const TRIGGER_TRANSACTION_CREATED = 'transaction_created';
+
+    const TRIGGER_TRANSACTION_UPDATED = 'transaction_updated';
+
+    const TRIGGER_MANUAL = 'manual';
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array<string>
+     */
+    protected $fillable = [
```

PHPStan (2.1.15)

PHPDoc type array<string> of property App\Models\Rule::$fillable is not covariant with PHPDoc type array<int, string> of overridden property Illuminate\Database\Eloquent\Model::$fillable.
You can fix 3rd party PHPDoc types with stub files:
   https://phpstan.org/user-guide/stub-files

(property.phpDocType)